src/main/resources/application.properties

```diff
@@ -31,3 +31,5 @@ spring.h2.console.enabled=true
 jwt.header=Authorization
 jwt.secret=secret
 jwt.expiration=7200
+
+allow.origin=*
```

Comment:
Wasn't this changed to the domain somewhere else?

Reply:
Not that I am aware of and the issue #6 is still open. With the changes included in this PR, the domain in the Access-Control-Allow-Origin header should now be able to be defined and adjusted after compilation with the application.properties. This covers issue #6.